scipy/interpolate/_interpolate.py

```diff
@@ -113,256 +111,23 @@ def lagrange(x, w):
 
 class interp2d:
     """
-    interp2d(x, y, z, kind='linear', copy=True, bounds_error=False,
-             fill_value=None)
+    `interp2d` was deprecated in SciPy 1.10 and has been removed as of
+    SciPy 1.12.0.
```

Comment:
Can change this from 12 to 13 now.

Reply:
In fact, I'd consider 1.14 if 1.13 is going to be an out-of-order release to catch up with numpy 2.0. 
What's the plan for 1.13 @tylerjereddy, is it expected around February 2024 or in June 2024?

Reply:
`1.13.0` is expected go out as soon as NumPy `2.0.0` is out, but I'm not sure if the schedule for `2.0.0` is being bumped beyond early February or not.